updated readme

Lukeghenco opened the pull request:
Adding more accurate ES6 styling for functions, fixed the reducer action that wasn't setting the state back to 0 correctly and cleaned up some odd spacing and unneeded semicolons. @aturkewi @PeterBell

Lukeghenco committed:
updated readme

Adding more accurate ES6 styling for functions, fixed the reducer action that wasn't setting the state back to 0 correctly and cleaned up some odd spacing and unneeded semicolons.

aturkewi commented on README.md:
```diff
@@ -20,27 +22,30 @@ Next, let's define a reducer function in `reducers/counterReducer.js` to pass to
+      return state - 1
     case 'RESET_COUNT':
-      return 0;
+      state = 0
```
Why set `state = 0` here? This shouldn't have to be here I don't think. Just returning 0 should modify the state in the store like in all the other cases in this switch

aturkewi commented on README.md:
```diff
@@ -49,19 +54,19 @@ Let's start our server by using `npm start`. Visit http://localhost:8080 in your
 import { createStore } from 'redux';
 import counterReducer from './reducers/counterReducer';
```
This section is to 'play around in console' but you cannot use `import` in the console. It seems we want to tell people to add these `console.log()` and `store.dispatch()` calls to `index.js` instead and refresh the page while watching the console.

Lukeghenco replied:
good catch. I'll update the readme for this one.

Lukeghenco replied:
Finished

Lukeghenco committed:
remove state = 0

Lukeghenco committed:
updated README to mention src/index.js

This now mentions that we should be just console.logging stuff from our index.js file

Lukeghenco commented:
@aturkewi changes have been made and ready for merge
